### DISCUSSION TITLE
CACHE_SECONDS env doesn't seem to work???

### DISCUSSION THREAD

**Discussion by kokofixcomputers** [anuraghazra/github-readme-stats#3949] [Q&A] [answered]
The CACHE_SECONDS environmental variable doesn't seem to work when value is set to 1 but works when value is set to 0. Am i doing something wrong? or is it a bug?

**Comment by kokofixcomputers** [comment]
When i set it to 10 it doesn't refresh after 10 seconds.

**Comment by kokofixcomputers** [comment]
or at least the /status/pat-info remaining count doesn't go down.

**Comment by qwerty541** [comment, marked as answer]
Hello @kokofixcomputers 

Recently I have merged a pull request https://github.com/anuraghazra/github-readme-stats/pull/4539 which made the server to return correct cache disabling headers when the `CACHE_SECONDS` environment variable set to 0. You can merge the recent changes in master branch into your fork and enjoy the card updating on each request.

Previously it also worked, but only in case you are accessing your card directly in separate browser window. When you observing the card in your profile, it was served from `camo.githubusercontent.com`, GitHub reuploads all images to their CDN and looks like clamping the minimum cache timing.
